immuni_common/helpers/utils.py

```diff
@@ -52,3 +53,32 @@ def dense_dict(dictionary: Dict) -> Dict:
         for k, v in dictionary.items()
         if v is not None
     }
+
+
+# Note: This could be a Generic[T] with T as TypeVar. That would allow
+#  us to define a WeightedPair[str] and have the weighted_random
+#  return a valid typed str when used, but generic NamedTuples are not yet
+#  supported by mypy:
+#  https://github.com/python/mypy/issues/685
```

Comment:
add one space to the first line or remove one from the others :D

Reply:
Why? It's an indentation meant to make it clear that it's the same comment. We also do this in the TODOs